### ISSUE TITLE
APT package

### ISSUE THREAD

**Issue by elcomtik** [cronie-crond/cronie#77]
Just curious why there is no APT package?

I saw some in Debian experimental repository, but no other. I would really appreciate having cronie for Debian/Ubuntu systems.

**Comment by t8m** [comment]
I do not think this place is where you could get an answer. I am afraid you'd have to ask Debian crond maintainer.

**Comment by elcomtik** [comment]
In the beginning, I thought that you create packages also. I will ask them
